roles/prereqs/tasks/main.yml

```diff
@@ -1,17 +1,24 @@
 ---
 
-# This role installs the prerequisites of RKE2 on control plane
-# (aka server) nodes and on worker (aka agent)   nodes that are
-# RHEL/CentOS 7/8 based.
-# TODO: Add Ubuntu 18/20 support
-
 - name: Populate service facts
   ansible.builtin.service_facts:
 
 - name: Gather the package facts
   ansible.builtin.package_facts:
     manager: auto
 
+- name: Is the images tarball in the tarball_install/ diretory
+  stat:
+    path: "{{ playbook_dir }}/tarball_install/rke2-images.linux-amd64.tar.zst"
+  register: got_images
+  delegate_to: 127.0.0.1
```

Comment:
I'd run this as non-root if possible. It might be fair to think the playbook is not run as a root user.